modules/op-creating-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -132,6 +132,42 @@ For advanced CEL-based filtering while using {pac}, you can use the following fi
 * `event_title`: Matches the title of the event, such as the commit title for a `push` event, and the title of a pull or merge request for a `pull_request` event. Currently, only GitHub, Gitlab, and Bitbucket Cloud are the supported providers.
 * `.pathChanged`: A suffix function to a string. The string can be a glob of a path to check if the path has changed. Currently, only GitHub and Gitlab are supported as providers.
 
+In addition, you can access the full payload as passed by the Git repository provider. Use the `headers` field to access the headers of the payload, for example, `headers['x-github-event']`. Use the `body` field to access the body of the payload, for example, `body.pull_request.state`.
+
+:FeatureName: using the header and body of the payload for CEL-based filtering with {pac}
+include::snippets/technology-preview.adoc[]
+
+In the following example, the pipeline run starts only if all of the following conditions are true:
+
+* The pull request is targeting the `main` branch
+* The author of the pull request is `superuser`
+* The action is `synchronize`; this action triggers when an update occurs on a pull request
```

Comment:
Per the ISG, periods at the end of full sentences in a list. All three need a period.